.github/actions/lint-front-matter/rules/updated-is-current.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @param {string} file Filename of the file being linted.
+ * @param {Object} yaml The yaml front matter from the file.
+ * @param {?string} yaml.updated The updated property in the yaml front matter.
+ * @param {Array<string>} failures Error messages from failed lint checks.
+ */
+module.exports = (file, {updated}, failures) => {
+  // If updated is undefined, just return early.
+  // Rather than error here, let another lint rule that explicitly checks for
+  // the presence of the updated property handle it.
+  if (!updated) {
+    return;
+  }
+
+  const today = new Date();
+  today.setUTCHours(0, 0, 0, 0);
```

Comment:
I think this will actually hurt folks who live far west—places like the US west coast. UTC could be a day ahead if you make a post ... before 8am. I know this is obscure, but still.

My brain is a bit confused with time math, but I think you want to pass... +12 here. (The farthest timezone _west_).

Reply:
Yeah I think we may actually scrap this per the discussion with pete.

If `updated` was added in a file, i'm going to check that it's greater than `date`.
If `updated` was modified in a file, I'm going to check that it's greater than the existing file's `updated`.